docs/_static/llama-stack-spec.html

```diff
@@ -1528,15 +1528,15 @@
                 ]
             }
         },
-        "/v1/health": {
+        "/v1/inference/health": {
```

Comment:
Wondering if it would be better to keep the `/health` endpoint and have it indicate all API's health status including inference services. This would also keep the backward compat on client SDKs. 

Otherwise I assume we would need `<api>/health` for all APIs for consistency.

Reply:
Thanks for your inputs!

Actually, that wasn’t intentional - it happened after running ./docs/openapi_generator/run_openapi_generator.sh. I don’t plan on removing "/v1/health".

I had considered using inspect/health or placing health under inspect/providers. However, I feel that /inspect might eventually become a dedicated admin API, making it inaccessible to users. Given that, /v1/inference/health seems like a natural fit and many APIs follow this pattern for resource-specific health checks.

Reply:
+1 @yanxi0830 I think _if_ we do this, we should do this at the `/inspect/` level. Re: "dedicated admin API", I think we can structure it so that regular users can get some information whereas authenticated administrators can get all information about the system. That way the API isn't walled off from normal users (developers.)